.github/workflows/deploy-standalone-plugins.yml

```
@@ -0,0 +1,63 @@
+name: Deploy standalone plugins to WordPress.org
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'release/**'
+    paths:
+      - '.github/workflows/deploy-standalone-plugins.yml'
+  pull_request:
+    branches:
+      - trunk
+      - 'release/**'
+      - 'feature/**'
+    paths:
+      - '.github/workflows/deploy-standalone-plugins.yml'
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  release:
+    name: New Release
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set matrix
+        id: set-matrix
+        # Load the JSON file and parse from "{name: {slug, version}, ...}" to "include: [{ name, slug, version }, ...]"
+        # for use in the matrix.
+        run: echo "matrix="$(jq -c '{include:[keys[] as $k | {name:$k,slug:.[$k].slug,version:.[$k].version }]}' plugins.json) >> $GITHUB_OUTPUT
+  deploy:
+    needs: release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJSON(needs.release.outputs.matrix) }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js (.nvmrc)
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+      - name: Install npm dependencies
+        run: npm ci
+      - name: Building standalone plugins
+        run: npm run build-plugins
+      - name: Deploy Standalone Plugin - ${{ matrix.slug }}
+        #uses: 10up/action-wordpress-plugin-deploy@stable
```

I used same `deploy.sh` that used in `10up/action-wordpress-plugin-deploy@stable` action.

---

Can you please add a code comment explaining that inline so that it's captured for any future reviewers of this code?
```suggestion
        # The `deploy.sh` script called below is almost a copy of the one used in `10up/action-wordpress-plugin-deploy@stable`.
```

---

Or are we able to actually use the original `10up/action-wordpress-plugin-deploy@stable` action directly? Is this file just a temporary change, or is it meant as the actual solution to merge?

---

Never mind, @joemcgill clarified that we want to eventually use the actual 10up GitHub workflow.

Something we need to make sure here is that we remove the temporary workflow triggers _before_ we switch to that action, to avoid accidentally publishing the WebP Uploads plugin from this PR.